pkg/corerp/frontend/controller/applications/testdata/graph-app-httproute-in.json

```diff
@@ -0,0 +1,55 @@
+[
+    {
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/httpRoutes/sql-rte",
+        "name": "sql-rte",
+        "properties": {
+            "application": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/Applications/myapp",
+            "provisioningState": "Succeeded"
+        },
+        "type": "Applications.Core/httpRoutes"
+    },
+    {
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/sql-app-ctnr",
+        "name": "sql-app-ctnr",
+        "properties": {
+            "application": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/Applications/myapp",
+            "connections": {
+                "sql": {
+                    "source": "/planes/radius/local/resourcegroups/default/providers/Applications.Datastores/sqlDatabases/sql-db"
+                }
+            },
+            "provisioningState": "Succeeded",
+            "status": {
+                "outputResources": {
+                    "id": "/some/thing/else",
+                    "localId": "something"
+                }
+            }
+        },
+        "type": "Applications.Core/containers"
+    },
+    {
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/sql-ctnr",
+        "name": "sql-ctnr",
+        "properties": {
+            "application": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/Applications/myapp",
+            "container": {
+                "ports": {
+                    "web": {
+                        "port": 8080,
+                        "protocol": "TCP",
+                        "provides": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/httpRoutes/sql-rte"
```

Comment:
Do we still need this provides for non-httproute scenario? @nithyatsu

Reply:
@rynowak what do you think below?

> Do we still need this `provides` field for non-httproute scenario? @nithyatsu

Reply:
As of public release, we had decided to retain code support for httproutes, since it would facilitate customers who were already using them but removed them from docs ( @willtsai took care of this decision and effort).